packages/desktop-client/src/components/modals/ImportTransactions.jsx

```diff
@@ -335,33 +343,124 @@ function Transaction({
   flipAmount,
   multiplierAmount,
   categories,
+  onCheckTransaction,
+  reconcile,
 }) {
   const categoryList = categories.map(category => category.name);
   const transaction = useMemo(
     () =>
-      fieldMappings
+      fieldMappings && !rawTransaction.isMatchedTransaction
         ? applyFieldMappings(rawTransaction, fieldMappings)
         : rawTransaction,
     [rawTransaction, fieldMappings],
   );
 
-  const { amount, outflow, inflow } = parseAmountFields(
-    transaction,
-    splitMode,
-    inOutMode,
-    outValue,
-    flipAmount,
-    multiplierAmount,
-  );
+  let amount, outflow, inflow;
+  if (rawTransaction.isMatchedTransaction) {
+    amount = rawTransaction.amount;
+    if (splitMode) {
+      outflow = amount < 0 ? -amount : 0;
+      inflow = amount > 0 ? amount : 0;
+    }
+  } else {
+    ({ amount, outflow, inflow } = parseAmountFields(
+      transaction,
+      splitMode,
+      inOutMode,
+      outValue,
+      flipAmount,
+      multiplierAmount,
+    ));
+  }
 
   return (
     <Row
       style={{
         backgroundColor: theme.tableBackground,
+        color:
+          (transaction.isMatchedTransaction && !transaction.selected_merge) ||
+          !transaction.selected
+            ? theme.tableTextInactive
+            : theme.tableText,
       }}
     >
+      {reconcile && (
+        <Field width={31}>
+          {!transaction.isMatchedTransaction && (
+            <Tooltip
+              content={
+                !transaction.existing && !transaction.ignored
+                  ? 'New transaction. You can import it, or skip it.'
+                  : transaction.ignored
+                    ? 'Already imported transaction. You can skip it, or import it again.'
+                    : transaction.existing
+                      ? 'Updated transaction. You can update it, or import it again instead, or skip it.'
```

Comment:
nit.  Can this be reworded to "You can update it, import it again, or skip it."

Reply:
Done, I updated the tooltip.